appium-examples/Cucumber/emusim/Rakefile

```diff
@@ -0,0 +1,49 @@
+def run_tests(deviceName, platformName, platformVersion, app, junit)
+  system("deviceName=\"#{deviceName}\" platformName=\"#{platformName}\" platformVersion=\"#{platformVersion}\" app=\"#{app}\" parallel_cucumber features -n 20")
```

Comment:
you... shouldn't need the quotes, these are setting environment variables, which... isn't the ideal way to do this either, but will work for now. :)
```
system "deviceName=#{deviceName} platformName=#{platformName} platformVersion=#{platformVersion} app=#{app} parallel_cucumber features -n 20"
```

Reply:
sounds good.